resources/cdk/aws-doc-sdk-examples-javascriptv3-usecases-transcribe-demo/README.md

```diff
@@ -0,0 +1,104 @@
+# AWS Code Examples Repository - AWS CDK App Template
+
+Use the template files in this folder to create the unique source code and the associated AWS CloudFormation template file for your finished AWS CDK app.
+
+## Prerequisites
+
+To set up your development machine in order to customize and run this AWS CDK template app, following the instructions for TypeScript in the "Prerequisites" section of the [Getting started with the AWS CDK](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html) topic in the *AWS CDK Developer Guide*.
+
+## Files
+
+This folder contains the following template files:
+
+* ``.gitignore``: A list of files and folders that are generated as you create your AWS CDK app but should not be checked in to the AWS Code Examples Repository.
+* ``cdk.json``: Information that the AWS CDK needs in order to find the corresponding AWS CDK app.
+* ``package.json``: Information that Node Package Manager (NPM) needs, such as package names, dependencies, scripts, and versions.
+* ``README.md``: This file.
+* ``setup.ts``: Programmatic information about the AWS CDK app.
+* ``tsconfig.json``: Information that the TypeScript compiler needs, such as root level files and various compiler options.
+
+## Customization
+
+To use these template files to create your AWS CDK app, modify the ``setup.ts`` file to include the AWS resources that you want the AWS CDK to generate. For more information, see the [AWS CDK Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide).
+
+To download the necessary dependent packages in order to run your AWS CDK app, first run the following command from within this folder:
+
+``npm install``
+
+To generate an AWS CloudFormation template file that represents your AWS CDK app logic, run the following commands from within this folder:
+
+For Windows: ``rmdir /s /q cdk.out``
+
+For Mac/*nix: ``rm -rf cdk.out``
+
+(This makes sure that the ``cdk.out`` folder is regenerated when you run ``cdk synth`` in the next command)
+
+``cdk synth > TranscribeApp.yaml``
+
+To run the AWS CDK app, which creates a corresponding stack in AWS CloudFormation and which in turn creates the specified AWS resources, run the following command from within this folder:
+
+``cdk deploy``
+
+To destroy the AWS CDK app, which deletes the corresponding stack from AWS CloudFormation and which in turn deletes the specified AWS resources, run the following command from within this folder:
+
+``cdk destroy``
+
+## Merging
```

Comment:
Should this section be removed for those simply wanting to use/experiment with the CDK app?

Reply:
Done